admin/sql/updates/20180331-mbs-9664-non-loop-checks.sql

```diff
@@ -0,0 +1,17 @@
+\set ON_ERROR_STOP 1
```

Comment:
Just needs adding to upgrade.json too. Shall we run it in production now (if you haven't already)?

Reply:
Did both, thanks!